doc/source/ray-air/examples/torch_detection.ipynb

```diff
@@ -0,0 +1,1081 @@
gKeNDERGp5I52CxQxpAaNPBpNZV3c4WywNxj9hzxNV+YhSEv/3iRzRR6WIEIE/shrKuOODV0YDntzZXIHz0tfXx5gqxmmBTcn0CyDyJhuQnW3klvXnjrxVQbDyZVkeHBwoCsaEIfbCaIfCCt2cF3hl3nvoLgxcTaKeA/IABdfJWdaDbIiSSAIOqOvLy8uDg4PT09NTU1NDQ0MyERXTwuN+x8bG2jZxlE9ATwlDDK6oRm4MVJwCAjyMjo6iMkYgj2aZLlvnggtOrSNXhVYWlu9iy+3t7ZFEUgzAmHjvOUEYMShqlEyFC/GaxPrqvWmb06qaRY33+/v73nuQ2GDlPTIwUdSeCEq0rz6eYhVCAFMWt99bXVHmqbBaVmnTmQW5YAieiGJClLrqrGamz67ToohYL+aVxVmOinKQOHjRhtZfEtO0Ign1JtKGHxImKGemwC2LxDRKgzKFJOBX5Z+xFF0bwaEnwWLyFRTc9a7CRhizLNVqVU3Jzphtir65PIXkmVHEVF53UeSoJZJ9SW1+qZZaQboOXhaxzeQvEwMftLAEdOPj45/73Oempqb+4A/+4C//8i8rpjSJc2JTsuPJo71RRxTv5MaLVbEd4dYygsu5BZY0Maq4Nz01BRmlyXbjHblUVh64TEata2xxDWstIs5fYVx7PevcpvAIIQmRSlfsEuIfucyf+hqsjzxK/LDcUwmrN/kKqlyVSuWVV1559913a5HAUWIAvY86sV00GEsv9jZyWol/rVaDiYi5KQ04TdO01WqRkZMXcirVH1+W5fj4eJZl29vbAN+lDY+OQw2tWLwU3iAmb7Vfac8VRcGGOTg4uHv37tjY2Pnz52dmZrCVRMQQeFZXV8n2vPU5i0wpSBBuDDtcHB7S6NJ6tdhL8OXVVcuQDRaNW+PysOzOGBls3cIE0RStUhuj39tFKuLIUXiDyEkXYEbiD9R/XticCZoSVAclVi4ixLgwAXPl6HG0JFv6sWuX7Q+RSJNMlfZNYtJXbJSutbmjpXB0dCQh0IqNVH0ikNHnJBF5Ts/+CZMtb68AChSoNOUdbW72XzfquShM7kf2VFZbBz4xKaLU5DAJxlVZ9ZEwpK4wMZENZVgiufM4KyYbracuaMIbDvDw4UPn3MnJyZUrV0jKsJLMvshszmdhHaqCQWWeEivquiikTSN9fMWnHdN9fSI2dJFfDBEUEKzmw7u0OCRzSZL8+q//+sjIyPe+970QUZVyGxiWGg8hmFJQZgM5g1EvXJRn9PX1EZ0pK2IrKrGrWK+cfIBAOSIvuD2Jqf1QW8bEQ1cFxRKzlm9RIOnMVetxy/c8YbXLqMzgzhJpYrv8U602PzEikURZvD4t/qIkSZALHx8ff/bZZ9977z0ZghAptjrr89KpIU7s2qjlrnHJg1HuEtMUlIqGPARBIoaDvnkcMBdZq9VarRZhzcHBAQUkHiU4G9svs2aOcJa0pzPurJRVWiENkLqnp2djY2NlZWVkZGRqampxcXF+fj5JkomJiaWlpZ6entXV1d3d3Z2dHYb8YQdw87gojkyn06HFmv2f2RQEvlHkK+dcvV53Ni5qf39/Z2cHJqXgB2ftr3wXEQz2lPgaiEZN4HlUUHXOjY6OspjyHMSapYmLldHkDM44y459xygBongjqsSHK1ibZ2ZN/nH0EELwf/fv/t3XX389hAAvdWJiggMDmV2+l1czGtF7j3J/bpOFE9P1Lo2nSXriTdxVqYQOf2xcQgRqK5zk0+SL0mhEpIu0F0I0lyCLROO8VboSwwQx5VxDMEBc1x+bOVnMYEF9ZlMZdQ6DcSQSY6wnNlbYG4092AgObxCq3j41NfXCCy9MT0/Pzs6qaKZG2djbpVb4JSKLU5Y4FSitU6MoClhcnPM8akrWo5R/lU33horgsFXAdCaImBgDb2tr6/XXX//GN77RbDbBRiDYlYbps7ljn6TD3I1ElbViZUT50H25yOsrJpBdcxZMsfvxK5g/AFwwGagOqRGTUyvbJlELlbN+WkWdsZ2V5RXW9EScHlvn+HSlptySWilS+1DRd/xEtKv1Ry0IUbYu3p3FgrQsSTSSRTwKIVG6o8zaLILNHYUh45xrNpty1Xj00ipb+/v79Xod8gkRFRsD/AqLGQ8L4ycxqo8zBEkHzVnDmjfAkyerjeG9HxkZmZ+fB7EZGxu7ePEiXPK7d+/euHGDZALpLu29ig2/dBEtQntGjzgOgBQ3gNi0Wi3k6eky1RNRLVrWI4uYLbJRpTGLSutaQOFjcHCQ0SJsRRUyE2Puo7tXmPwUvYdJkqDPQ6tU1SYkxyFCiDAG5TH8/Sf0VdB2OXlZFkXiqvCUhquyZHEDnrZszTQznQl7aaUU7pVR+xJxtyJ6mTCZ+FjMXq64ME2V+PmlNigdc+ks2lXkqNOLyYaa0jU+cvztmek+s+dU7pCB0FHn2pTPeu8r1tutu05NwePg4GBra+v27dtZlmmCR7fbRRBGRqdryqUsIIsvoNNbbO6t2Ue4EyAsaIk/CwXIfimB1Qqnxs4ED+HAC71xziGSU61Wf+7nfm5ubu6//Jf/8ujRI8rvAkOkxZaaXoewL3aFdh5buYyGg6cR4VVPKr54WX9tAO6x0+moSkRkJKw5NTlAIjK2sYxOiKCDPFJoiVOlJ+xp7If82fhdNjTeQv6peapxOB/vRn4y66LwEeOAh1U1VcUyKszGhdZgyH5iFJoYM3zicYNX8CF8C/Ec6IFqhrlN/syyjI6HJElarZZeoMipVqvFR8w95R2dJeKK5QHN5fZUUGX3bm5u7u7u3r9/f2ho6OLFi1tbW7Ozs1euXPlH/+gftdvtW7duvfrqq8vLy2maTk5Oip4vKSf2JAByEcm2BBNjkXEUDjE+Pt7X17ewsFAUxc7OztbWFmOvZUB4mYtUNKomwwVMzXnkYtic+/v75GEw+mVgSxMgcVaNCJH2NQVOCtGplVj1cHmIsVfjObbPSpBmMtM0o6fGL6bpkVNH+xNUSmGI2t
```

Comment:
"open the images for the corresponding annotation and add the image as a column to our dataset"